app/bundles/CoreBundle/Assets/js/1a.content.js

```diff
@@ -1442,6 +1471,49 @@ Mautic.activateLiveSearch = function (el, searchStrVar, liveCacheVar) {
             mQuery(btn + ' i').removeClass('ri-eraser-line').addClass('ri-search-line');
         }
     }
+
+    /**
+     * Keyboard navigation for Global Search modal.
+     * Supports: ArrowUp, ArrowDown, Tab, Enter, Escape
+     */
+    if (mQuery(el).attr('id') === 'globalSearchInput') {
+        mQuery(el).off('keydown.globalSearchNav').on('keydown.globalSearchNav', function (e) {
+            const items = mQuery('#globalSearchResults .gsearch--results-item');
+            if (!items.length) return;
+
+            const key = e.key || e.keyCode;
+
+            if (key === 'ArrowDown' || key === 40 || key === 'Tab') {
+                e.preventDefault();
+                activeIndex = (activeIndex + 1) % items.length;
+            } else if (key === 'ArrowUp' || key === 38) {
+                e.preventDefault();
+                activeIndex = (activeIndex - 1 + items.length) % items.length;
+            } else if (key === 'Enter' || key === 13) {
+                const link = items.eq(activeIndex).find('a').get(0);
+                if (link) link.click();
```

Comment:
```suggestion

                if (link) {
                    link.click();
                }
```
nitpick: Once again, readability.